src/components/molecules/ScheduleItemNG/ScheduleItemNG.scss

```diff
@@ -24,6 +27,19 @@ $schedule-info-block-width: 60px;
     }
   }
 
+  &__icon {
+    flex-shrink: 0;
+    height: $event-icon--size;
+    width: $event-icon--size;
+    margin-left: $event-icon--margin;
+    margin-right: $event-icon--margin;
```

Comment:
```suggestion
    margin: 0 $spacing--xs;
```

Reply:
could we use `$spacing--xs` to avoid extra var inside the .scss file?

Reply:
yeah, nice catch. 🙃 thanks!
refactored here: https://github.com/sparkletown/sparkle/pull/1990/commits/75872581539be7979b443a4afa553d1b6dc7a520